src/client/pages/Embark/ErrorModal.tsx

```diff
@@ -11,6 +11,11 @@ import { ErrorModal, ErrorHeading, ErrorText } from 'components/ErrorModal'
 const ButtonContainer = styled.div`
   display: flex;
   justify-content: flex-start;
+  padding-top: 1rem;
```

Comment:
The changes in this file has nothing to do with the Summary/Quote Details section, right? Maybe create a separate PR for this? :)